.github/workflows/after-merge.yml

```diff
@@ -0,0 +1,157 @@
+# Run after merge to trunk
+on:
+  push:
+    branches: [master] # $default-branch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['14.x', '16.x']
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      # 'yarn install' must be done at the top level, to build all the
+      # cross-package symlinks
+      - name: yarn install
+        run: yarn install
+      - name: check if git is dirty
+        run: git diff --stat --exit-code
+      - name: check dependencies
+        run: yarn check-dependencies
+      # 'yarn build' loops over all workspaces
+      - name: yarn build
+        run: yarn build
+
+      - uses: ./.github/actions/save-node
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: notify on failure
+        if: failure()
+        uses: ./.github/actions/notify-status
+        with:
+          from: ${{ secrets.NOTIFY_EMAIL_FROM }}
+          to: ${{ secrets.NOTIFY_EMAIL_TO }}
+          password: ${{ secrets.NOTIFY_EMAIL_PASSWORD }}
+
+  dev-canary:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # note: only use one node-version
+        node-version: ['14.x']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/restore-node
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      # Adapted from https://johnny.sh/notes/publish-canary-lerna-cicd/
+      - name: configure NPM token
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: check credentials
+        run: npm whoami
+      - name: publish to NPM dev tag
+        run: |
+          yarn lerna publish --conventional-prerelease --canary --exact \
+            --dist-tag=dev --preid=dev-$(git rev-parse --short=7 HEAD) \
+            --no-push --no-verify-access --yes
+      - name: notify on failure
+        if: failure()
+        uses: ./.github/actions/notify-status
+        with:
+          from: ${{ secrets.NOTIFY_EMAIL_FROM }}
+          to: ${{ secrets.NOTIFY_EMAIL_TO }}
+          password: ${{ secrets.NOTIFY_EMAIL_PASSWORD }}
+
+  coverage:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # note: only use one node-version
+        node-version: ['14.x']
```

Comment:
with only one node version, why use the matrix strategy?

Reply:
So that the job has the node version string in it.  I like it that way. :)